jest.config.js

```diff
@@ -1,26 +1,29 @@
 /** @type {import("ts-jest/dist/types").InitialOptionsTsJest} */
 module.exports = {
-  preset: "ts-jest",
-  moduleNameMapper: {
-    "^(\\.{1,2}/.*)\\.js$": "$1",
-  },
-  testEnvironment: "node",
-  coveragePathIgnorePatterns: [
-    "./src/internal/queries/",
-    "./src/types/generated",
-    "./tests/e2e/ans/publishANSContracts.ts",
-  ],
-  testPathIgnorePatterns: ["dist/*", "examples/*"],
-  collectCoverage: true,
-  setupFiles: ["dotenv/config"],
-  coverageThreshold: {
-    global: {
-      branches: 50, // 90,
-      functions: 50, // 95,
-      lines: 50, // 95,
-      statements: 50, // 95,
-    },
-  },
-  // To help avoid exhausting all the available fds.
-  maxWorkers: 4,
+	preset: "ts-jest",
```

Comment:
please use space indent instead of tabs, and please align your linter with existing file, thanks!